game/assets/jak1/text/game_text_en.gs

```diff
@@ -8,12 +8,16 @@
         "CAMERA OPTIONS")
 (#x1001 "NORMAL"
         "NORMAL")
-(#x1002 "INVERTED"
-        "INVERTED")
-(#x1003 "HORIZONTAL CAMERA CONTROL"
-        "HORIZONTAL CAMERA CONTROL")
-(#x1004 "VERTICAL CAMERA CONTROL"
-        "VERTICAL CAMERA CONTROL")
+(#x1002 "FLIPPED"
+        "FLIPPED")
+(#x1003 "FIRST-PERSON HORIZONTAL CAMERA CONTROL"
+        "FIRST-PERSON HORIZONTAL CAMERA CONTROL")
+(#x1004 "FIRST-PERSON VERTICAL CAMERA CONTROL"
+        "FIRST-PERSON VERTICAL CAMERA CONTROL")
+(#x1005 "THIRD-PERSON HORIZONTAL CAMERA CONTROL"
+        "THIRD-PERSON HORIZONTAL CAMERA CONTROL")
+(#x1006 "THIRD-PERSON VERTICAL CAMERA CONTROL"
+        "THIRD-PERSON VERTICAL CAMERA CONTROL")
```

Comment:
See if you can find a way to shorten this text. Lines on the progress menu should not break (be 2 lines large) at 4x3.

In the `post` for `progress-normal` (the overriden one) you can find the draw functions and change the parameters to make the text slighty smaller if you need to.